.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
```

Comment:
Is this sentence? incomplete

Reply:
That's the default value. It does seem a little cryptic...maybe because these templates are more oriented towards open source projects with a larger community, or they have a looser definition of "bug"?